front/app/containers/Admin/communityMonitor/components/LiveMonitor/components/CommunityMonitorUpsell/index.tsx

```diff
@@ -57,7 +57,16 @@ const CommunityMonitorUpsell = () => {
             </Button>
           </Tooltip>
 
-          <Button buttonStyle="text" textColor={theme.colors.tenantPrimary}>
+          <Button
+            buttonStyle="text"
+            textColor={theme.colors.tenantPrimary}
+            onClick={() => {
+              window.open(
+                'https://support.govocal.com/en/articles/10981781-community-monitoring-resident-satisfaction-survey',
```

Comment:
Clicked this link and getting a 404

Reply:
Yes Nola mentioned to me that they hadn't added anything to the article yet, but this would be the link it will be available at :+1: